elements/rh-secondary-nav/demo/analytics.html

```diff
@@ -0,0 +1,209 @@
+<link rel="stylesheet" href="../demo.css">
+<link rel="stylesheet" href="../../rh-secondary-nav-lightdom.css">
+<script type="module" src="../rh-secondary-nav.js"></script>
+
+<div class="mock-nav"></div>
+<rh-secondary-nav role="navigation" data-analytics-region="Ansible secondary nav">
+  <a href="#" slot="logo" id="logo-id">
+    Red Hat Ansible Automation Platform
+  </a>
+  <ul slot="nav" aria-labelledby="logo-id">
+    <li>
+      <rh-secondary-nav-dropdown>
+        <a href="#" data-analytics-level="1" data-analytics-text="Explore" slot="link">Explore</a>
+        <rh-secondary-nav-menu slot="menu">
+          <rh-secondary-nav-menu-section>
+            <h2 slot="header" id="ansible-platform">
+              <a href="#" data-analytics-level="2" data-analytics-text="Why Red Hat Ansible Automation Platforms" data-analytics-category="Explore">
```

Comment:
@Rickey Messick - looking at the demo, these header links are in the same layout region as the ul links that follow. Should their `*-category=""` values also be `Explore|Why Red Hat Ansible Automation Platforms` vs just `Explore`? I would suggest they should, but this is consistent with the current RHDC main nav. (See "Services & support" for an example in production.)

Reply:
agree value for the category should be 'Explore|Why Red Hat Ansible Automation Platforms'

Reply:
Got it! I updated this one, and also the 'additional features' heading to match this logic.